src/taskStatusManager.js

```diff
@@ -0,0 +1,15 @@
+import toDoList from './todolist.js';
+
+export default function updateListItem(status, index) {
+  toDoList.updateStatus(status, index);
+}
+
+document
+  .getElementById('clear-completed')
+  .addEventListener('click', () => {
+    const elements = document.getElementsByClassName('task-status');
+    for (let i = elements.length - 1; i >= 0; i -= 1) {
+      if (elements[i].checked) elements[i].parentElement.remove();
+    }
+    toDoList.removeCompletedTasks();
+  });
```

Comment:
I think using forEach would make the code better and shorter and would increase efficiency,
which a part of the Javascript best practices